documentation/sphinx/source/api-c.rst

```diff
@@ -460,6 +460,20 @@ An |database-blurb1| Modifications to a database are performed via transactions.
 
    |future-returnvoid|
 
+.. function:: FDBFuture* fdb_database_create_snapshot(FDBDatabase* database, uint8_t const* snapshot_command, int snapshot_command_length)
+
+   Create a snapshot of the database.
+
+   ``snapshot_command``
+         A pointer to all the snapshot command arguments.
+
+         In particular, if the original ``fdbcli`` command is ``snapshot <arg1> <arg2> <argN>``, then the string ``snapshot_command`` points to is ``<arg1> <arg2> <argN>``, which including the spaces.
```

Comment:
```suggestion
         In particular, if the original ``fdbcli`` command is ``snapshot <arg1> <arg2> <argN>``, then the string ``snapshot_command`` points to is ``<arg1> <arg2> <argN>``.
```

Reply:
I _think_ users will want access to the snapshot UID, so we should probably return something equivalent to `Future<UID>`

Reply:
Done. Thanks